service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDetailsHelper.java

```diff
@@ -61,6 +64,16 @@ public static CaseDetails removeTemporaryFields(CaseDetails caseDetails, Class c
         return caseDetails;
     }
 
+    public static Map<String, Object> nullifyTemporaryFields(Map<String, Object> caseDetails, Class clazz) {
```

Comment:
clazz should be "class"

Reply:
@tomaszpowroznik directed me to - https://stackoverflow.com/questions/2529974/why-do-java-programmers-like-to-name-a-variable-clazz/2529980

after consideration this is not required to change